easybuild/easyblocks/w/wps.py

```diff
@@ -275,8 +288,12 @@ def run_wps_cmd(cmdname, mpi_cmd=True):
                 # GEOGRID
 
                 # setup directories and files
-                for d in os.listdir(os.path.join(tmpdir, "geog")):
-                    os.symlink(os.path.join(tmpdir, "geog", d), os.path.join(tmpdir, d))
+                if mainver < 4:
```

Comment:
Same here, please use LooseVersion to check versions

Reply:
addressed by https://github.com/andreas-h/easybuild-easyblocks/pull/4